sopel/cli/run.py

```diff
@@ -112,18 +112,24 @@ def signal_handler(sig, frame):
 
 
 def add_legacy_options(parser):
+    # TL;DR: option -d/--fork is not deprecated.
+    # When the legacy action is replaced in Sopel 8, 'start' will become the
+    # new default action, with its arguments.
+    # The option -d/--fork is used by both actions (start and legacy),
+    # and it has the same meaning and behavior, therefore it is not deprecated.
     parser.add_argument("-d", '--fork', action="store_true",
-                        dest="daemonize", help="Daemonize Sopel")
+                        dest="daemonize",
+                        help=("Daemonize Sopel."))
```

Comment:
It's weird that this has nested parentheses even though it's only one line. (It's also weird that this section of the code uses nested parentheses for `help` argument values when they're not needed, but changing that would require touching code that's not related to documenting the configuration.)

Maybe revert this hunk and just add the `.` in `help`?

Reply:
```suggestion
                        help="Daemonize Sopel.")
```